source/robot_model/include/robot_model/Model.hpp

```diff
@@ -73,19 +73,24 @@ class Model {
                                                        unsigned int frame_id);
   
   /**
-   * @brief It checks if the vector's elements are inside at their limits
+   * @brief Checks if the vector's elements are inside the limits in parameter
    * @param vector the vector to check
+   * @param lower_limits the lower bounds of the limits
+   * @param upper_limits the upper bounds of the limits
    * @return true if all the elements are inside at their limits, false otherwise.
    */
   bool in_range(const Eigen::VectorXd& vector, const Eigen::VectorXd& lower_limits, const Eigen::VectorXd& upper_limits);
 
   /**
-   * @brief It clamps the vector's elements according to their limits
+   * @brief Clamps the vector's elements according to the limits in parameter
```

Comment:
Imperative form would be `Check if ...`, same with `Clamp the vector...`.

Also I think `inside the parameter limits` or `according to the parameter limits` would be preferable to `in parameter`

Reply:
Always good to have someone who lived in England in the team :D